[1,"Error occurred during cleanup: Error: El proceso no tiene acceso al archivo porque est√° siendo utilizado por otro proceso. (os error 32): remove 'C:\\Users\\vales\\AppData\\Local\\Temp\\quarto-sessiona4f876c69666331c'\n"]
